src-tauri/src/mocks/init/init_edge_map.rs

```diff
@@ -0,0 +1,123 @@
+use crate::aux_data_structures::neighbor_info::{Neighbor, NeighborInfo};
+use std::collections::HashMap;
+
+pub fn init_edge_map(packets: Vec<NeighborInfo>) -> HashMap<(u32, u32), (f64, u64)> {
```

Comment:
Since this is in src/mocks does it mean there will be another function that creates edges from real data? In other words, do you intend to use this function only for fake data? If not, why is it in this directory?

Reply:
You're right, that's a strange directory placement. Would it make more sense to change `mocks` module to a parent module of `constructors`, with submodules `mock` and `init`?

I had `init` functions for both maps in case we wanted to generate mock packets, populate hashmaps, and use those to create a graph. But since it makes more sense to add edges to our existing nodes at runtime, they're a little superfluous for the moment.

Reply:
> Would it make more sense to change `mocks` module to a parent module of `constructors`, with submodules `mock` and `init`?
This makes much more sense I think.

Reply:
Changed. Current file structure is: 
`constructors`
   - `init`
      - contains init functions for graph and edge and node hashmaps. Intended to allow us to run a full run of algorithms from the packets up
  - `mocks`
     - contains functions for generating random lists of `NeighborInfo` and `MeshNode` packets
     - contains stub for creating random edges from an existing hashmap of `MeshNode` structs (our location info)